packages/farcaster-js-hub-rest/package.json

```diff
@@ -13,6 +13,7 @@
   "license": "MIT",
   "dependencies": {
     "@farcaster/core": "^0.14.8",
+    "neverthrow": "^6.0.0",
```

Comment:
I was having issues when this version was the latest 6.2, which did not match that of @farcaster/core